### PR TITLE
Make diktat own code consistent with code style

### LINKED ISSUE #218: Make it easier to load custom json config when running diktat as maven or gradle plugin
Currently we use file from jar, because we cannot specify location in plugin configuration.

Also, it will allow to set maximum line length and other options for diktat self check.

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Code style formatting
* Changed antrun config to allow custom rules-config.json (closes #218 )

## Fixme
All issues labeled `diktat check` and some others are blocking this.
Including: ~#215~, ~#216~, #217, ~#218~ ...

### CODE REVIEW

File: rules-config.json
Comment (akuleshov7) [inline rules-config.json:1]: may be delete it? and @aktsay6 will add yml?

**Review by akuleshov7 — APPROVED** [review]
lgtm

File: rules-config.json
Comment (petertrr) [reply rules-config.json:1]: I can wait until he adds yml, merge it into this branch and only then merge this pull request.

**Pull request merged**
